### PR TITLE
Create exploit for CVE-2015-2993 (sysaid admin account creation)

### PR DESCRIPTION
**Pull request by pedrib**
This pull request creates a module for CVE-2015-2993, which creates an administrator account on SysAid Help Desk 14.4 and below. This module has been tested on Windows and Linux.

### CODE REVIEW

File: modules/auxiliary/admin/http/sysaid_admin_acct.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        },
+      'Author' =>
+        [
+          'Pedro Ribeiro <pedrib[at]gmail.com>' # Vulnerability discovery and MSF module
+        ],
+      'License' => MSF_LICENSE,
+      'References' =>
+        [
+          [ 'CVE', '2015-2993' ],
+          [ 'OSVDB', 'TODO' ],
```
Comment (void-in) [inline modules/auxiliary/admin/http/sysaid_admin_acct.rb]: Please see https://github.com/rapid7/metasploit-framework/pull/5474#issuecomment-108731648

**Comment by wchen-r7** [comment]
Delayed label applied. [Author pinged](https://github.com/rapid7/metasploit-framework/pull/5473#issuecomment-112560324).

**Comment by pedrib** [comment]
Sorry for the delay! Modified it as requested.

**Comment by jvazquez-r7** [comment]
I'm downloading the beta from the home page. If the beta isn't vulnerable anymore would be nice if you could provide us with installer URL if available or just a PCAP for verification (without SSL pls, otherwise a ssl capable proxy would be good enough :))

**Comment by jvazquez-r7** [comment]
After downlaoding the beta, it is 15.2 :\ I cannot find older installers at the home page. If there is any link, feel free to share and we can deploy a testing environment. Otherwise pcap's will be needed.

**Comment by pedrib** [comment]
@jvazquez-r7 I've uploaded the vulnerable version to dropbox:
https://www.dropbox.com/s/fbkydxrcbfl9j0y/SysAidServer64.exe?dl=0 (Windows)
https://www.dropbox.com/s/k0zr201ef1kfts3/sysaid-server-linux.tar.gz?dl=0 (Linux)

These are the vulnerable 14.2 version. Let me know if you need anything else!

**Comment by jvazquez-r7** [comment]
Thanks @pedrib , landed! Tested successfully:

```
msf auxiliary(sysaid_admin_acct) > set TARGETURI /
TARGETURI => /
msf auxiliary(sysaid_admin_acct) > run
  1 Land #5470, @pedrib's module for SysAid CVE-2015-2993

[+] 172.16.158.131:8080 - Created administrator account with credentials msf:password
[*] Auxiliary module execution completed
msf auxiliary(sysaid_admin_acct) > creds
Credentials
===========

host            service          public  private   realm  private_type
----            -------          ------  -------   -----  ------------
172.16.158.131  8080/tcp (http)  msf     password         Password

```

Then I verified the account manually.

I just did minor cleanup, see final result here: https://github.com/rapid7/metasploit-framework/commit/787c0e2c418c73d9390ed5386335c0194b221408

**Pull request merged**